scripts/qa_crowbarsetup.sh

```diff
@@ -1198,20 +1204,17 @@ EOF
 
 function onadmin_installcrowbarfromgit()
 {
-    do_installcrowbar \
-        "CROWBAR_FROM_GIT=1
-            /opt/dell/bin/install-chef-suse.sh --from-git --verbose"
+    if iscloudver 5plus ; then
```

Comment:
I guess, that is a long story, that nobody told me yet. kkaempf did some controversial things there.